contracts/jobs/JobsManager.sol

```diff
@@ -432,7 +432,7 @@ contract JobsManager is ManagerProxyTarget, IVerifiable, IJobsManager {
         // Slashing period must be over for claim
         require(claim.endSlashingBlock < block.number);
 
-        uint256 fees = claim.segmentRange[1].sub(claim.segmentRange[0]).add(1).mul(job.maxPricePerSegment);
+        uint256 fees = JobLib.calcFees(claim.segmentRange[1].sub(claim.segmentRange[0]).add(1), job.transcodingOptions, job.maxPricePerSegment);
```

Comment:
Do you think `calcFees` should be invoked in the `job()` transaction and just saved on the job itself, rather than calculated in three separate places?

Reply:
essentially calculating fee/segment for the job? On the other hand I could see why this wouldn't work because you don't know which renditions will end up being claimed, and only if the claim contains all renditions would the pre-calculated fees make sense.